shopinvader_product_sale_packaging/views/product_packaging.xml

```diff
@@ -4,14 +4,11 @@
         <field name="model">product.packaging</field>
         <field
             name="inherit_id"
-            ref="sale_by_packaging.product_packaging_tree_view_inherit"
-        />
-        <field
-            name="groups_id"
-            eval="[(4, ref('shopinvader.group_shopinvader_manager'))]"
+            ref="sell_only_by_packaging.product_packaging_tree_view_inherit"
         />
         <field name="arch" type="xml">
             <field name="force_sale_qty" position="after">
+                <!-- TODO: group on this field was shopinvader.group_shopinvader_manager-->
                 <field name="shopinvader_display" />
```

Comment:
Both fields added in this view were restricted to `shopinvader.group_shopinvader_manager` group. Now this group moved to `shopinvader_restapi`, but our addon shouldn't depend on `shopinvader_restapi` just for a security group...